src/assignment/views/AssignmentDetail.tsx

```diff
@@ -53,6 +52,7 @@ import { getAssignmentContent } from '../assignment.helpers';
 import { AssignmentLayout, AssignmentRetrieveError } from '../assignment.types';
 
 import './AssignmentDetail.scss';
+import { FragmentList } from '../../collection/components';
```

Comment:
Import order

Reply:
fixed in latest bundle branch: feature/AVO2-374-duplicate-bundle-and-collection